src/black/parsing.py

```diff
@@ -148,24 +148,29 @@ def lib2to3_unparse(node: Node) -> str:
 
 
 def parse_single_version(
-    src: str, version: Tuple[int, int]
+    src: str, version: Tuple[int, int], type_comments: bool
```

Comment:
```suggestion
    src: str, version: Tuple[int, int], *, type_comments: bool
```
Let's make it keyword-only